content/200-concepts/100-components/01-prisma-schema/06-relations/07-referential-actions.mdx

```diff
@@ -43,12 +43,12 @@ Referential actions are a preview feature. To enable the `referentialActions` pr
 1. [Upgrade to Prisma version 2.26.0](../../../../guides/upgrade-guides/upgrading-to-latest).
 2. Add `referentialActions` to the `previewFeatures` field of the Prisma Client [`generator`](../../../../reference/api-reference/prisma-schema-reference#generator) block in the `prisma.schema` file:
 
-  ```prisma file=schema.prisma
```

Comment:
investigating this…

Reply:
This is indented so the schema is under the list item - might be that mdx requires 3 (or 4) spaces here for this to work in the first place, and prettier assuming this should just be on the top level instead because it is not 3 spaces.

Reply:
@janpio you are right, 2 spaces get removed, but 3 or 4 works

https://user-images.githubusercontent.com/4734297/124757297-d9d12800-df46-11eb-97c7-fcdb2494edd2.mp4

Reply:
So the behavior of VSCode and Co to by default indent with 2 spaces does not put it "into the list" in Markdown, which is what we want in the end. That explains _a lot_ of errors we had in the past I think about code blocks "escaping" the list item they were meant to be in.

Reply:
> That explains a lot of errors we had in the past I think about code blocks "escaping" the list item they were meant to be in.

That's why I love the Prettier, before using it I thought this is the valid code:

```js
function Hello(props) {
  return 
    <div>{props.propWithReallyLongName}</div>
}
```